autoload/lsp/utils.vim

```diff
@@ -70,7 +70,7 @@ if has('win32') || has('win64')
     endfunction
 else
     function! lsp#utils#uri_to_path(uri) abort
-        return s:decode_uri(a:uri[len('file://'):])
+        return s:decode_uri(substitute(a:uri, '\v^file:/*', '/', 'g'))
```

Comment:
If the uri contains hostname, the result should NOT be expected. This expect /path/to/the/file but the result which have hostname is /hostname/path/to/file.